client/src/Components/Widget0.js

```diff
@@ -62,18 +62,23 @@ class Widget0 extends React.Component {
      }
  
      trackData(){
- 
-         setInterval(function(){
+        var self = this;
+        setInterval(function(){
              fetch('/api/trackleads')
              .then(res => res.json())
              .then(function(data){
 
-                 let {mapStyle} = this.state;
+                 let {mapStyle} = self.state;
 
                  var trackleaders = data.trackleaders_aggregate_feed.trackleaders_feed;
                  var pointData ;
                  var points = [];
                  
+                 var geojsonWrapper = {
+                    "type": "FeatureCollection",
+                    "features": []
+                    }  
+
                  trackleaders.forEach(trackleader => {
                     pointData = {
                         type: 'Point',
```

Comment:
Notice that the "type" here should be "feature", and then the "feature" should contain "geometry" and "property" keys for valid geojson.  See https://docs.mapbox.com/mapbox-gl-js/example/multiple-geometries/ for an example